src/vconsole/vconsole-setup.c

```diff
@@ -234,38 +225,78 @@ static void setup_remaining_vcs(int fd, int vc_max, bool utf8) {
                 return;
         }
 
+        fontbuf = new(unsigned char, 32*32*512/8);
```

Comment:
hmm, i think it would be nicer to write `cfo.width * cfo.height * cfo.charcound / 8` or so, as that's mor descriptive of what's going on...